heron/cli/src/python/args.py

```diff
@@ -14,7 +14,7 @@
 ''' args.py '''
 import os
 
-import heron.common.src.python.utils as utils
+import heron.common.src.python.utils.config as utils
```

Comment:
agree on this. I understand it's exhausting to fix issues caused by renaming, but please fix them.